Makefile

```diff
@@ -31,7 +31,7 @@ all: check generate_docs
 
 # Run all available linters
 .PHONY: check
-check: check_shell check_python check_golang check_terraform check_docker check_base_files test_check_headers check_headers check_trailing_whitespace
```

Comment:
why have you removed it?

Reply:
Now we use standard containers for tests, so we don't need tests for Dockerfiles